review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
```

Comment:
Token would be a more accurate description here regarding how it's used
```suggestion
 * @param {string} authorization Authorization token to authenticate the user
```